src/state/wallet/hooks.ts

```diff
@@ -100,25 +115,30 @@ export function useTokenBalance(account?: string, token?: Token): TokenAmount |
   return tokenBalances[token.address]
 }
 
+const EMPTY_CURRENCIES: any[] = []
+const EMPTY_TOKENS: any[] = []
+const EMPTY_ACCOUNT: any[] = []
```

Comment:
All these can be combined to a single EMPTY_ARRAY. It will work the same

Reply:
I prefer separate them for meaningful and ease to read

Reply:
For me an EMPTY_ARRAY is crystal clear. I see EMPTY_TOKENS, I then need to check for whatever it is.

For EMPTY_ARRAY, no more step is needed